src/Console/Command/CommandHelp.php

```diff
@@ -288,7 +288,15 @@ public static function getDisplayableAllowedValues(FixerOptionInterface $option)
                 return !is_callable($value);
             });
 
-            sort($allowed);
+            $toString = function ($value) {
+                return call_user_func('PhpCsFixer\Console\Command\CommandHelp::toString', $value);
+            };
+            usort(
+                $allowed,
+                function ($valueA, $valueB) use (&$toString) {
+                    return strcasecmp($toString($valueA), $toString($valueB));
```

Comment:
Agreed, this is a hacky workaround for PHP 5.3 that does not allow calling `self::foo()` in closures.